force-app/main/default/labels/CustomLabels.labels-meta.xml

```diff
@@ -2786,7 +2786,7 @@
         <language>en_US</language>
         <protected>true</protected>
         <shortDescription>Status Automation introduction</shortDescription>
-        <value>These settings automate changing the Recurring Donation Status when donors miss installment payments.&lt;br/&gt;&lt;br/&gt;Using these settings, NPSP updates the Recurring Donation&apos;s Status when the current date exceeds the Last Donation Date plus the future expected installment date and the Number of Days for Lapsed or Number of Days for Closed.&lt;br/&gt;&lt;br/&gt;For example, given a monthly Recurring Donation started on January 1st, and Number of Days for Lapsed is set to 10 days, the Recurring Donation becomes Lapsed on February 10th if no installment payment is received.&lt;br/&gt;&lt;br/&gt;Elevate Recurring Donations are not included in this automation.</value>
+        <value>These settings automate changing the Recurring Donation Status when donors miss installment payments.&lt;br/&gt;&lt;br/&gt;For example, given a monthly Recurring Donation started on January 1st, and Number of Days for Lapsed is set to 10 days, the Recurring Donation becomes Lapsed on February 10th if no installment payment is received. If Number of Days for Closed is set to 20 days, the Recurring Donation becomes Closed on February 20th.&lt;br/&gt;&lt;br/&gt;Elevate Recurring Donations are not included in this automation.</value>
```

Comment:
The RD will become Lapsed after 10 days have passed, which is on February 12th. (2/1 + 10 days + 1 days). The same applies to the Closed RD scenario.

Reply:
Ok, right. Thanks for clarifying. So we don't start counting the 10 days until after the 1st of the month has passed (Feb 2nd in this case). Which is why the status change is on the 12th. I'll re-write this and post a new version for review.

Reply:
@andrewyu-salesforce - I may reword this a bit, but is the following technically correct?

For example, given a monthly Recurring Donation started on January 1st, and Number of Days for Lapsed is set to 10 days, the Recurring Donation becomes Lapsed on February 12th if no installment payment is received. If Number of Days for Closed is set to 20 days, the Recurring Donation becomes Closed on February 22th.

To explain further, the lapsed Status update occurs on the 12th because Salesforce begins counting Number of Days for Lapsed on February 2nd (the day after the expected installment date). Then, after 10 days, the Status update occurs on February 12th.

Reply:
That's correct, 2/12 and 2/22 are the date to mark RD as Lapsed/Closed. I did the math wrong in the first reply. We give one extra day because we want to give the whole threshold days before changing the RD status.

Reply:
Ok, thanks for confirming my understanding. I'll make another label update and ping you to review and merge.

The special case you mentioned makes sense to me. It seems in line with how other RDs would be treated.